File: app/src/main/java/com/intelegain/agora/model/SaveSkillMatrixEntity.java
package com.intelegain.agora.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by maninder on 2/1/18.
 */

public class SaveSkillMatrixEntity {
    @SerializedName("UserID")
    @Expose
    public String userID;
    @SerializedName("EmpId")
    @Expose
    public String empId;
    @SerializedName("lstEmpSkill")
    @Expose
    public List<SkillMatrixData.EmpSkill> lstEmpSkill = null;
}

File: app/src/main/java/com/intelegain/agora/model/SaveSkillMatrixEntity.kt
package com.intelegain.agora.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.intelegain.agora.model.SkillMatrixData.EmpSkill

/**
 * Created by maninder on 2/1/18.
 */
class SaveSkillMatrixEntity {
    @SerializedName("UserID")
    @Expose
    var userID: String? = null
    @SerializedName("EmpId")
    @Expose
    var empId: String? = null
    @SerializedName("lstEmpSkill")
    @Expose
    var lstEmpSkill: List<EmpSkill>? = null
}